lib/helper/generateBackfillTableHelper.py

```diff
@@ -0,0 +1,138 @@
+import os
+import sys
+import json
+from ..common.logConfig import get_logger
+from lib.common.commonUtil import NetworkUtil
+from lib.common.commonUtil import CommonUtil
+from archiveMozillaHelper import ArchiveMozillaHelper
+from perfherderDataQueryHelper import PerfherderDataQueryHelper
+
+
+logger = get_logger(__name__)
+
+
+class GenerateBackfillTableHelper(object):
+    DEFAULT_HG_QUERY_REVISION_JSON_URL = "https://hg.mozilla.org/mozilla-central/json-rev"
+    DEFAULT_BACKFILL_TABLE_LOCAL_FN = "backfill_table.json"
+
+    @staticmethod
+    def generateArchieveRevisionRelationTable(input_history_backfill_table, input_backfill_days=14, input_backfill_repo="mozilla-central", input_app_name="firefox", input_channel_name="nightly"):
```

Comment:
modify the method name to `abc_xyz_foo_bar` will better

Reply:
this is the official method for another python to called, so should be named it more meaningful

Reply:
`generate_archieve_revision_relation_table()`